icons/lockedCornerFlag.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="35px" height="35px" viewBox="0 0 35 35" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>icon-locked-corner-flag</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Artboard" transform="translate(-107.000000, -31.000000)">
+            <g id="icon-locked-corner-flag" transform="translate(107.000000, 31.000000)">
+                <polygon id="Rectangle" fill="#333333" points="0 0 35 0 35 35"></polygon>
```

Comment:
I think this polygon will need to be combined into a single path in your SVG editor for this to display correctly. Did you check out what this looked like in storybook?

Reply:
for our icon setup, it only handles square 24x24 single color icons. Maybe this one shouldn't go in this repo and just stay in your app until we handle multicolor svgs